cub/cub/device/dispatch/kernels/transform.cuh

```diff
@@ -258,18 +265,36 @@ _CCCL_DEVICE void transform_kernel_vectorized(
 
   auto provide_array = [&](auto... inputs) {
     // load inputs
-    // TODO(bgruber): we could support fancy iterators for loading here as well (and only vectorize some inputs)
-    [[maybe_unused]] auto load_tile_vectorized = [&](auto* in, auto& input) {
-      auto in_vec    = reinterpret_cast<const load_store_t*>(in);
-      auto input_vec = reinterpret_cast<load_store_t*>(input.data());
-      _CCCL_PRAGMA_UNROLL_FULL()
-      for (int i = 0; i < load_store_count; ++i)
+    _CCCL_PDL_GRID_DEPENDENCY_SYNC();
+
+    [[maybe_unused]] auto load_tile = [&](auto in, auto& input) {
```

Comment:
Q: What scoped variables are we pulling in? 

I only see constexpr variables like `load_store_count` or inputs

Reply:
None, capture list emptied